Modules/Remote/VkFFTBackend.remote.cmake

```diff
@@ -0,0 +1,50 @@
+#-- # Grading Level Criteria Report
+#-- EVALUATION DATE: 2022-04-06
+#-- EVALUATORS: [Tom Birdsong]
+#--
+#-- ## Compliance level 5 star (AKA ITK main modules, or remote modules that could become core modules)
+#--   - [ ] Widespread community dependance
+#--   - [ ] Above 90% code coverage
+#--   - [X] CI dashboards and testing monitored rigorously
+#--   - [X] Key API features are exposed in wrapping interface
+#--   - [ ] All requirements of Levels 4,3,2,1
+#--
+#-- ## Compliance Level 4 star (Very high-quality code, perhaps small community dependance)
+#--   - [X] Meets all ITK code style standards
+#--   - [ ] No external requirements beyond those needed by ITK proper
+#--   - [ ] Builds and passes tests on all supported platforms within 1 month of each core tagged release
+#--            - [ ] Windows Shared Library Build with Visual Studio
+#--            - [ ] Mac with clang compiller
+#--            - [ ] Linux with gcc compiler
```

Comment:
Can we check off any of these?

Reply:
Sounds reasonable to me. I will make the change. I am also chasing down CMake bugs related to building VkFFT dependencies with OpenCL.